Makefile

```diff
@@ -66,9 +66,18 @@ production:
 	$(eval export KEY_VAULT=s146p01-kv)
 	$(eval export AZURE_SUBSCRIPTION=s146-getintoteachingwebsite-production)
 
+.PHONY: review_aks
+review_aks: test-cluster
+	$(if $(PR_NUMBER), , $(error Missing environment variable "PR_NUMBER", Please specify a pr number for your review app))
+	$(eval include global_config/review_aks.sh)
+#	$(eval export PR_NAME=review-get-into-teaching-app-${PR_NUMBER})
```

Comment:
Hi... Please take out the hashed lines if not needed in the nearest future

Reply:
removed